php-nginx/whitelist_functions.php

```diff
@@ -14,22 +14,33 @@
     $whitelist = array_filter(array_map('trim', explode(',', $whitelist)));
 }
 
-$suhosinBlacklist = ini_get('suhosin.executor.func.blacklist');
 $disable_functions = ini_get('disable_functions');
 
+// It's tricky, but we still don't have suhosin enabled and we can not fetch
+// the list from our suhosin config. Now it defines the list as a literal.
+// Note: We need to sync this list with our default suhosin config
+// (deb-package-builder/extensions/suhosin/debian/ext-suhosin.ini).
+// TODO: Find a better way
```

Comment:
Why can't we do (in `entrypoint.sh`):

1. Enable Suhosin
2. run `whitelist_functions.php`
3. run `composer run-script`

I think the reason is if suhosin is already enabled, `whitelist_functions.ini` won't be loaded? But in that case we can re-enable suhosin, which is not great, but probably better than hardcoding the ini values. Thoughts?